vnext/src/RNTester/js/examples-win/DatePicker/DatePickerExample.windows.tsx

```diff
@@ -6,7 +6,7 @@
 
 import React = require('react');
 import {Text, View} from 'react-native';
-import {DatePicker} from '../../../../Libraries/Components/DatePicker/DatePicker';
+const {DatePicker} = require('react-native-windows');
```

Comment:
As is I think this means we lose typings.

The relative paths are related to a tsconfig issue that popped up during the 0.62 upgrade, that ended up just not getting fixed after. Need to look up the details, but IIRC it's not hard to make paths to RNW work correctly. We're eventually wanting to remove Typescript components + RNTester pages though in favor of flow, so I'm not sure anyone has put the effort in.